float32 fl_steer_ang
float32 fr_steer_ang
float32 rl_steer_ang
float32 rr_steer_ang

float32 fl_wheel_rpm
float32 fr_wheel_rpm
float32 ml_wheel_rpm
float32 mr_wheel_rpm
float32 rl_wheel_rpm
float32 rr_wheel_rpm

bool    short_fl_wheel
bool    short_fr_wheel
bool    short_ml_wheel
bool    short_mr_wheel
bool    short_rl_wheel
bool    short_rr_wheel

bool    hand_brake
